content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst

```diff
@@ -1,94 +1,121 @@
-=========================
-Create Bills of Materials
-=========================
+================================
+Create a Bill of Materials (BoM)
+================================
 
-A *Bill of Materials* is a document defining the quantity of each
-component required to manufacture a finished product. It also includes
-the routing and individual steps of the manufacturing process.
+A *Bill of Materials* (or *BoM*) is a document that defines the quantity of each component required
+to manufacture or deliver a finished product. It can also include various operations types and
+steps, and instructions for the individual step guidelines needed to complete a production process.
 
-With Odoo, you can link multiple BoMs to each product and use it to
-describe multiple variants of them. Each BoM will, yet, be associated
-with one product only.
+In Odoo Manufacturing, multiple :abbr:`BoMs (Bills of Materials)` can be linked to a product, so
+that even product variants can have their own specific :abbr:`BoMs (Bills of Materials)`.
 
-This feature will help optimize your manufacturing process while saving
-you time.
+Correctly setting up a Bill of Materials helps optimize the manufacturing process, and saves time.
 
-Setting up a BoM
-================
+Set up a Bill of Materials (BoM)
+================================
 
-You can use BoMs without routings. You will use this if you choose to
-manage your manufacturing operations using manufacturing orders only. In
-other words, you choose to realize your manufacturing process in one
-step and do not track the steps the product goes through.
+:abbr:`BoMs (Bills of Materials)` can be set up with or without operations steps or instructions,
+and can include as many (or as few) components as needed. The simplest setup is one without
+operations or instructions, in which case the production is solely managed with *Manufacturing
+Orders*.
 
-Before creating your *BoM*, you have to create the product using the
-*BoM* and, at least, one of the components. Go to the :menuselection:`Master Data menu --> Products`
-and create both the finished product and the component.
+To create a :abbr:`BoM (Bill of Materials)`, navigate to :menuselection:`Manufacturing -->
+Products --> Bills of Materials`, and click :guilabel:`Create`. Then, in the :guilabel:`Product`
+field, choose the product that the :abbr:`BoM (Bill of Materials)` will be associated with.
```

Comment:
To create a :abbr:`BoM (Bill of Materials)`, navigate to :menuselection:`Manufacturing -->
Products --> Bills of Materials`, and click :guilabel:`Create`. Then, in the :guilabel:`Product`
field, choose the product that will be associated with the :abbr:`BoM (Bill of Materials)`.